cdap-app-fabric/src/main/java/co/cask/cdap/metadata/ExistingEntitySystemMetadataWriter.java

```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-package co.cask.cdap.data.tools;
+package co.cask.cdap.metadata;
```

Comment:
Using ExistingEntitySystemMetadataWriter.java to upgrade System metadata dataset, removed other steps for system.